assembl/templates/views/messageColumn.tmpl

```diff
@@ -0,0 +1,42 @@
+
+<div class="real-time-updates tac hidden phl">
+    <button class="btn-real-time js_loadPendingMessages">
+        <span class="pendingMessage"></span>
+    </button>
+</div>
+
+<style type="text/css">
+  #message_column_<%= column.getNumericId() %> .js_agentName,
```

Comment:
Ooof. I thought this might be how MAP does it. Okay, well then, my previous comment on documenting that this action needs to be taken goes out the window. But this architecture goes out the window on V2 anyway.